services/slam/server_test.go

```diff
@@ -107,6 +147,30 @@ func TestServer(t *testing.T) {
 		test.That(t, extraOptions, test.ShouldResemble, map[string]interface{}{})
 	})
 
+	t.Run("working get pointcloud map stream function", func(t *testing.T) {
+		injectSvc.GetPointCloudMapStreamFunc = func(ctx context.Context, name string) (func() ([]byte, error), error) {
+			reader := bytes.NewReader(pcd)
+			serverBuffer := make([]byte, chunkSizeServer)
+			f := func() ([]byte, error) {
+				n, err := reader.Read(serverBuffer)
+				if err != nil {
+					return nil, err
+				}
+
+				return serverBuffer[:n], err
+			}
+
+			return f, nil
+		}
+
+		reqPointCloudMapStream := &pb.GetPointCloudMapStreamRequest{Name: testSlamServiceName}
+		mockServer := makePointCloudStreamServerMock()
+		err = slamServer.GetPointCloudMapStream(reqPointCloudMapStream, mockServer)
+		test.That(t, err, test.ShouldBeNil)
+		test.That(t, mockServer.rawBytes, test.ShouldResemble, pcd)
```

Comment:
Is it possible for line 170 & 171's outputs to differ from one another? If so, how?

Reply:
updated comments to reflect intent of each check